doc/using-energyplus-for-compliance/CMakeLists.txt

```diff
@@ -1,35 +1,22 @@
-  ${PROJECT_SOURCE_DIR}/doc/title.tex
-  ${PROJECT_SOURCE_DIR}/doc/using-energyplus-for-compliance/using-energyplus-for-compliance.tex
-  ${PROJECT_SOURCE_DIR}/doc/header.tex
-  ${PROJECT_SOURCE_DIR}/doc/using-energyplus-for-compliance/src/using-energyplus-for-compliance.tex
-  ${PROJECT_SOURCE_DIR}/doc/using-energyplus-for-compliance/src/introduction.tex
-  ${PROJECT_SOURCE_DIR}/doc/using-energyplus-for-compliance/src/introduction/background.tex
-  ${PROJECT_SOURCE_DIR}/doc/using-energyplus-for-compliance/src/introduction/compliance-objects.tex
-  ${PROJECT_SOURCE_DIR}/doc/using-energyplus-for-compliance/src/introduction/tabular-summary-reports.tex
-  ${PROJECT_SOURCE_DIR}/doc/using-energyplus-for-compliance/src/introduction/features-related-to-appendix-g-and-leed.tex
-  ${PROJECT_SOURCE_DIR}/doc/using-energyplus-for-compliance/src/introduction/features-related-to-california-title-24.tex
+  ${PROJECT_SOURCE_DIR}/title.tex
+  ${PROJECT_SOURCE_DIR}/using-energyplus-for-compliance/using-energyplus-for-compliance.tex
+  ${PROJECT_SOURCE_DIR}/header.tex
+  ${PROJECT_SOURCE_DIR}/using-energyplus-for-compliance/src/using-energyplus-for-compliance.tex
+  ${PROJECT_SOURCE_DIR}/using-energyplus-for-compliance/src/introduction.tex
+  ${PROJECT_SOURCE_DIR}/using-energyplus-for-compliance/src/introduction/background.tex
+  ${PROJECT_SOURCE_DIR}/using-energyplus-for-compliance/src/introduction/compliance-objects.tex
+  ${PROJECT_SOURCE_DIR}/using-energyplus-for-compliance/src/introduction/tabular-summary-reports.tex
+  ${PROJECT_SOURCE_DIR}/using-energyplus-for-compliance/src/introduction/features-related-to-appendix-g-and-leed.tex
+  ${PROJECT_SOURCE_DIR}/using-energyplus-for-compliance/src/introduction/features-related-to-california-title-24.tex
 )
 
 # to get the list of included images, go to the root of the document folder and:
 # grep -r '\\includegraphics' * | cut -d ':' -f 2 | rev | cut -d '{' -f 1 | rev | cut -d '}' -f 1 | sed 's/^/  /g' | sort -n -t'.' -k 1.14
 SET( INCLUDED_IMAGES
 )
 
-SET( SOURCE_FILENAME "using-energyplus-for-compliance" )
-SET( OUTPUT_FILENAME "UsingEnergyPlusForCompliance" )
-
-add_custom_command( OUTPUT ${PROJECT_BINARY_DIR}/doc/${OUTPUT_FILENAME}.pdf
-	COMMAND ${CMAKE_COMMAND} -DXELATEX=${XELATEX} -DINNAME=${SOURCE_FILENAME} -DOUTNAME=${OUTPUT_FILENAME} -DORIGINAL_CMAKE_SOURCE_DIR=${PROJECT_SOURCE_DIR} -DORIGINAL_CMAKE_BINARY_DIR=${PROJECT_BINARY_DIR} -DTEX_INTERACTION=${TEX_INTERACTION} -P ${PROJECT_SOURCE_DIR}/cmake/BuildDocumentation.cmake
-		    WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/doc/${SOURCE_FILENAME}
-		    DEPENDS ${INCLUDED_TEX} ${INCLUDED_IMAGES}
-                 )
-add_custom_target( zPDF_${OUTPUT_FILENAME}
-                   DEPENDS ${PROJECT_BINARY_DIR}/doc/${OUTPUT_FILENAME}.pdf
-                 )
-
-# Make top-level 'documentation' target depend on this one
-add_dependencies(documentation zPDF_${OUTPUT_FILENAME})
+CREATE_DOC_TARGET("using-energyplus-for-compliance" "UsingEnergyPlusForCompliance")
```

Comment:
I think there should be a setting to turn back on trailing newlines upon file saving.